cephalopod/factory.py

```diff
@@ -11,7 +11,7 @@
 # noinspection PyUnresolvedReferences
 from . import models  # registers db models
 
-login_manager = LoginManager()
+multipass = Multipass()
```

Comment:
This belongs in core.py, not here. Then you mostl ikely also won't have to use a local import e.g. when using it frontend.py.

The only reason why `login_manager` was in here is that it was used nowhere else.